web/styles/search.css

```diff
@@ -52,6 +52,10 @@
             opacity: 1;
         }
 
+        &:focus {
```

Comment:
We really ought to get into the habit of using `:focus-visible`. Is that a possibility here?